hack/generated/controllers/crd_networking_loadbalancer_test.go

```diff
@@ -0,0 +1,113 @@
+/*
+Copyright (c) Microsoft Corporation.
+Licensed under the MIT license.
+*/
+
+package controllers_test
+
+import (
+	"context"
+	"reflect"
+	"testing"
+
+	. "github.com/onsi/gomega"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	network "github.com/Azure/azure-service-operator/hack/generated/_apis/microsoft.network/v1alpha1api20200501"
+	"github.com/Azure/azure-service-operator/hack/generated/pkg/genruntime"
+	"github.com/Azure/azure-service-operator/hack/generated/pkg/testcommon"
+)
+
+func Test_LoadBalancer_CRUD(t *testing.T) {
+	t.Parallel()
+
+	g := NewGomegaWithT(t)
+	ctx := context.Background()
+	tc := globalTestContext.ForTest(t)
+
+	rg := tc.CreateNewTestResourceGroupAndWait()
+
+	// Public IP Address
+	sku := network.PublicIPAddressSkuNameStandard
+	publicIPAddress := &network.PublicIPAddresses{
+		TypeMeta: metav1.TypeMeta{
+			Kind: reflect.TypeOf(network.PublicIPAddresses{}).Name(),
+		},
+		ObjectMeta: tc.MakeObjectMetaWithName(tc.Namer.GenerateName("publicip")),
+		Spec: network.PublicIPAddresses_Spec{
+			Location: tc.AzureRegion,
+			Owner:    testcommon.AsOwner(rg.ObjectMeta),
+			Sku: &network.PublicIPAddressSku{
+				Name: &sku,
+			},
+			Properties: network.PublicIPAddressPropertiesFormat{
+				PublicIPAllocationMethod: network.PublicIPAddressPropertiesFormatPublicIPAllocationMethodStatic,
+			},
+		},
+	}
+
+	tc.CreateResourceAndWait(publicIPAddress)
+	defer tc.DeleteResourceAndWait(publicIPAddress)
+
+	// It should be created in Kubernetes
+	g.Expect(publicIPAddress.Status.Id).ToNot(BeNil())
```

Comment:
we should probably remove this check from this test

Reply:
Fixed